README.md

```diff
@@ -112,7 +117,7 @@ end
 Your answer:
 ```text
 
-Replace this with your answer
+Describe is describing the expected output of a particular function (or method in this case), context is describing the condition under which it is being tested (in this example, its literally a conditional).
 ```
```

Comment:
`Describe` doesn't indicate output. In indicates what exactly you're testing. In this scenario that's the `add_tenant` method.